ui/src/main/resources/Meeting/Code/MeetingSheet.xml

```diff
@@ -251,16 +251,23 @@
           &lt;div class='presentation'&gt;
             &lt;input type='hidden' name='sendNotification' value='true'&gt;
             &lt;input type='hidden' name='doc' value="$services.model.serialize($doc.documentReference, 'default')"&gt;
-            &lt;label for='additionalMessage'&gt;
-                $services.localization.render('contrib.meeting.notification.additionalMessage')
+            ; &lt;label for='additionalMessage'&gt;
+                $!services.icon.render('email') $services.localization.render('contrib.meeting.field.message')
+                &lt;span class="xHint"&gt;
+                  $services.localization.render('contrib.meeting.notification.additionalMessage')
+                &lt;/span&gt;
               &lt;/label&gt;
-            &lt;textarea name="additionalMessage" id="additionalMessage" cols='40' rows='4'&gt;&lt;/textarea&gt;
+            : &lt;textarea name="additionalMessage" id="additionalMessage" cols='40' rows='4'&gt;&lt;/textarea&gt;
             #if($doc.getValue('lastEmailSent'))
-              &lt;p&gt;$services.localization.render('contrib.meeting.field.lastEmailSent') &lt;b&gt;$doc.display('lastEmailSent')&lt;/b&gt;&lt;/p&gt;
+               &lt;label&gt;
+              &lt;span class="xHint"&gt;
+                $services.localization.render('contrib.meeting.field.lastEmailSent') &lt;b&gt;$doc.display('lastEmailSent')&lt;/b&gt;
+                &lt;/span&gt;
+              &lt;/label&gt;
             #end
-            &lt;span class='buttonwrapper'&gt;
+            &lt;div class='buttonwrapper'&gt;
```

Comment:
it would be better to not use the older classes, so .buttonwrapper and .button should be replaced with .btn .btn-primary

Reply:
So all I need to do is just to change these two classes, in this code ?

Reply:
https://getbootstrap.com/docs/3.3/css/#buttons